src/UI/examples/Image/licence_images.txt

```diff
@@ -0,0 +1,2 @@
+All images taken from https://stocksnap.io
+All photos on StockSnap fall under the Creative Commons CC0 license. That means you can copy, modify, distribute any photo on the site, even for commercial purposes, all without asking permission!
```

Comment:
Not currently, where should we display this message? As for the modals, I could mention the source of the image in the description of the lightbox page.